### PR TITLE
Revert "[READY] KA rework"

### PR DESCRIPTION
**Pull request by Takahiru**
Reverts the mining nerf

### CODE REVIEW

**Comment by Jc1469** [comment]
Not bothered trying it myself because I've been busy with signal tech memes instead, but I've not actually heard anything positive about the modular KA things from the few people that've actually said anything about it, so.. I agree with this.

**Comment by Super3222** [comment]
I’ve seen people use it in game. I don’t understand why it needs to be reverted.

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -16,7 +16,7 @@
 			return
 
 		if("stealth")
-			new /obj/item/weapon/gun/energy/crossbow(src)
+			new /obj/item/weapon/gun/energy/kinetic_accelerator/crossbow(src)
```
Comment (ShadowDeath6) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:19]: :+1:

**Comment by Jc1469** [comment]
I approve of deleting the empty megafauna/goliath.dm which somehow nobody questioned when this was merged initially.

**Comment by ghost** [comment]
reverting shit without good explanation will always get a thumbs down from me.

**Comment by Time-Green** [comment]
@Takahiru why do you want it reverted so badly? Elaborate and maybe I can improve on that

**Comment by drderp3635** [comment]
@time-green its a straight need that's rather quite boring. All it does is make you have to spend 2000 points to get a fucking super ka and removes perks of hypers. Any parts that are fundamentally different are pretty much unusable. Currently there is no point in using them past your first supply drop since the are outclassed by plasma cutters on almost all aspects.

**Comment by Time-Green** [comment]
HKA's were WAY to overpowered. You still have the 3x3 mining and some rarer ones from necropolis crates.

I'll try and add more parts then.

**Comment by Takahiru** [comment]
its a straight up nerf, and it takes 4000 points to get two KAs upgraded and its barely noticeable

**Comment by Time-Green** [comment]
Yeah, you'll need to use new tactics. And I know a big part of it is a nerf, but mining was TOO easy for everyone. You'd literally just get HKA's and destroy everything

**Comment by Takahiru** [comment]
Well i wouldn't mind at least a higher tier of parts that cost more.

**Comment by Time-Green** [comment]
That's reasonable. You have a name and a way to acquire them in mind?
Maybe just more expensive?

**Comment by ghost** [comment]
It's pathetic.

You removed decent KAs, instead of super KAs. Plasma cutters are now far superior.

Golems cannot get KAs anymore.

The parts themselves are utterly useless. You spend 4000+ points on a KA that's barely as effective as a hyper KA when a plasma cutter is far more effective.

You didn't make lavaland harder. You just made it more tedious.

You didn't even use the half-decent system that /tg/station implemented for their KAs either. At least theirs didn't suck fucking cock, even if they DID remove hyper KAs. You just made your own spaghetti code version that's barely functional and extremely stupid.

If you're going to do something with KAs, port /tg/'s version. This one is both useless and inefficient.

**Comment by Time-Green** [comment]
Since the beginning I've been trying to create original content for the server
There's always things to adjust and improve, but adam, not if you can't even properly criticice something.

Golems can print wide guards, wich allow them to wield them
All the super parts should be about 2000 points
Gonna toot my own horn here, the code works well, and even has pure performance procs build in. Even you could easily add new parts

If the parts are too weak, then it would be so much easier to adjust that, then to revert all of it

**Comment by drderp3635** [comment]
Wide guard is absolute ass. Why would you even make it fucking shake your screen and drop when you have something in your offhand with no statistical improvement?

**Comment by Time-Green** [comment]
Wait what?
I never made it shake

**Comment by ghost** [comment]
You made it require no offhand.

**Comment by Time-Green** [comment]
Wideguard is simply bigger and lets fat fingered fucks use it
I never intended it to be doublewield or shake the screen

**Comment by Jc1469** [comment]
Another thing for the list if this doesn't get reverted, someone had a vampiric core and their KA wasn't healing at all, I even checked vars after them shooting mobs.

**Comment by ghost** [comment]
also, @Time-Green since you missed what i said

GOLEMS CANNOT GET KAS ANYMORE.

Because they CAN'T PRINT THEM

**Pull request closed without merging**